aemedge/styles/styles.css

```diff
@@ -12,33 +12,33 @@
 /* Fall back to Arial for 72 Brand Variable
  Added to styles.css for early loading */
 
-@font-face {
-  font-family: '72 Brand Variable Fallback';
+ @font-face {
+  font-family: "72 Brand Variable Fallback";
```

Comment:
Be careful with these, we should use all the same format rules.

Reply:
followed existing indentation